<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Progress</class>
 <widget class="QWidget" name="Progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="KS::BR::ProgressIcon" name="m_icon" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="m_definition">
       <property name="text">
        <string>Security reinforcement is protecting your computer</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="m_note">
         <property name="text">
          <string>KylinSec Host Security Reinforcement Software Detects Risks in Advance to Ensure Asset Security</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_generateReport">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Generate report</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="m_progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>4</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="m_scan">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_return">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string>return</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KS::BR::ProgressIcon</class>
   <extends>QWidget</extends>
   <header>src/ui/br/progress-icon.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
